src/app/admin/usersTable/UsersTable.tsx

```diff
@@ -162,18 +162,9 @@ const UsersTable: React.FC = () => {
                     method: emailSearch,
                 },
             }),
+            buildUserRoleFilter(),
```

Comment:
does this mean `buildFilters` can be non-async anymore?

Reply:
yup